<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	font: 20pt &quot;Roboto&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="point">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>•</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equals">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 92, 92);
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exponent">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QLabel" name="Screen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="root">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>n√Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backspace">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pi">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QLabel" name="operatorDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="valueDisplay">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>seven</tabstop>
  <tabstop>eight</tabstop>
  <tabstop>nine</tabstop>
  <tabstop>four</tabstop>
  <tabstop>five</tabstop>
  <tabstop>six</tabstop>
  <tabstop>one</tabstop>
  <tabstop>two</tabstop>
  <tabstop>three</tabstop>
  <tabstop>zero</tabstop>
  <tabstop>point</tabstop>
  <tabstop>equals</tabstop>
  <tabstop>plus</tabstop>
  <tabstop>exponent</tabstop>
  <tabstop>backspace</tabstop>
  <tabstop>minus</tabstop>
  <tabstop>tan</tabstop>
  <tabstop>clear</tabstop>
  <tabstop>multiply</tabstop>
  <tabstop>sin</tabstop>
  <tabstop>root</tabstop>
  <tabstop>divide</tabstop>
  <tabstop>cos</tabstop>
  <tabstop>pi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
